examples/loaders/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I assume this is to work around issues with top-level config files that need to be `.js` but also need to be CJS?

Reply:
Yup, exactly